### PR TITLE
Keyboard fixes

### PR DESCRIPTION
**Pull request by smontanari1**
Fixes dependency on msvcrt by using a script which chooses between msvcrt and termios implementation of keyboard hit, tested and confirmed working on headless raspberry pi over ssh

#56
#1

### CODE REVIEW

File: src/fixate/ui_cmdline/cmd_line.py [hunk 1]
```diff
@@ -1,67 +1,61 @@
...
+import time
 import textwrap
-import traceback
 from queue import Empty
-from queue import Queue
 from pubsub import pub
-from pynput import keyboard
-import fixate.config
-from fixate.config import RESOURCES
+from .kbhit import KBHit
```
Comment (pazzarpj) [inline src/fixate/ui_cmdline/cmd_line.py]: I think this should be an absolute import
```python
from fixate.ui_cmdline.kbhit import KBHit
```

File: src/fixate/ui_cmdline/kbhit.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+'''
+A Python class implementing KBHIT, the standard keyboard-interrupt poller.
```
Comment (pazzarpj) [inline src/fixate/ui_cmdline/kbhit.py]: Include where you found this code snippet into this doc string.

File: src/fixate/ui_cmdline/cmd_line.py [hunk 3]
```diff
@@ -1,67 +1,61 @@
...
 from fixate.core.exceptions import UserInputError
+from fixate.core.common import ExcThread
+from fixate.config import RESOURCES
+import fixate.config
 
 cmd_line_queue = Queue()
 wrapper = textwrap.TextWrapper(width=75)
 wrapper.break_long_words = False
 
 wrapper.drop_whitespace = True
```
Comment (pazzarpj) [inline src/fixate/ui_cmdline/cmd_line.py]: This is all just a revert of the changes made during #1.
Approved because it should be identical functionality

Comment (smontanari1) [reply src/fixate/ui_cmdline/cmd_line.py]: Do you want these lines included? they are removed in another change you made

Comment (pazzarpj) [reply src/fixate/ui_cmdline/cmd_line.py]: Keep them removed. The comment was meant to be from the full revert of the changes that Kieran made in #1.

**Pull request merged**